network-api/networkapi/wagtailpages/migrations/0008_collection_fix.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.14 on 2021-05-17 22:53
+from django.db import migrations
+
+
+def delete_factory_collections(apps, schema_editor):
+    Collection = apps.get_model('wagtailcore', 'Collection')
+    Collection.objects.filter(depth=1).exclude(id=1).delete()
```

Comment:
I'd feel more comforatble if this grabbed every collection with `id` > 1, moved any images in side of it into the id=1 collection (if any), and then deleted each collection once empty, instead.